environment.conf

```diff
@@ -1,2 +1,2 @@
-modulepath          = site:modules:$basemodulepath
+modulepath          = site-modules:modules:$basemodulepath
```

Comment:
This is opposite of what bolt does.

Reply:
As in bolt places `modules` before `site-modules`?  

This change just renamed site to site-modules.  idk if we care much about the ordering.  I think it would only matter in rare scenarios

Reply:
Yeah my assumption is that this order isn't super important in most cases. Bolt uses the first entry on the modulepath to install to the puppetfile from so it has to be modules and it seems like it would be nice if the standard control repo behaved the same.

Reply:
It used to be that way: https://github.com/puppetlabs/control-repo/pull/44